reference/src/layout/scalars.md

```diff
@@ -48,17 +49,17 @@ They have the same layout as the [pointer types] for which the pointee is
 
 > **Note**: in the current Rust implementation, the layouts of `isize` and
 > `usize` determine the following:
-> 
-> * the maximum size of Rust _allocations_ is limited to `isize::MAX`.
+>
+> * The maximum size of Rust _allocations_ is limited to `isize::MAX`.
 >   The LLVM `getelementptr` instruction uses signed-integer field offsets. Rust
 >   calls `getelementptr` with the `inbounds` flag which assumes that field
 >   offsets do not overflow,
 >
-> * the maximum number of elements in an array is `usize::MAX` (`[T; N:
->   usize]`. Only ZST arrays can probably be this large in practice, non-ZST
->   arrays are bound by the maximum size of Rust values,
+> * The maximum number of elements in an array is `usize::MAX` (`[T; N: usize]`).
```

Comment:
This continues the sentence from the previous bullet. I think it was deliberately lower-case.